lib/matplotlib/transforms.py

```diff
@@ -725,50 +716,23 @@ def union(bboxes):
         """
         if not len(bboxes):
             raise ValueError("'bboxes' cannot be empty")
-
-        if len(bboxes) == 1:
-            return bboxes[0]
-
-        x0 = np.inf
-        y0 = np.inf
-        x1 = -np.inf
-        y1 = -np.inf
-
-        for bbox in bboxes:
-            points = bbox.get_points()
-            xs = points[:, 0]
-            ys = points[:, 1]
-            x0 = min(x0, np.min(xs))
-            y0 = min(y0, np.min(ys))
-            x1 = max(x1, np.max(xs))
-            y1 = max(y1, np.max(ys))
-
-        return Bbox.from_extents(x0, y0, x1, y1)
+        x0 = min(bbox.xmin for bbox in bboxes)
```

Comment:
I guess I should use `np.min` everywhere (which passes `nan` through, whereas builtin `min` drops them).  (At least this would be consistent with `bbox.xmin`.)